packages/common/driver-definitions/src/storage.ts

```diff
@@ -385,6 +385,32 @@ export interface IDocumentServiceFactory {
 	): Promise<IDocumentService>;
 }
 
+export interface ISocketStorageDiscoveryParts {
+	/**
+	 * Niche interface to allow odsp-driver to expose specific pieces necessary for /broadcast-signal
+	 * endpoint. Please do not use this for anything else. We will phase this out once we know how
+	 * the interface for a more uniform interface across multiple drivers looks.
+	 */
+
+	// SPO gives us runtimeTenantId, we remap it to tenantId
```

Comment:
The host (Loop) wants it to be generic across any service and for the component using this to be unaware of which driver it's using. So I've introduced this extra interface so that Trello component doesn't have to specifically pull in odsp-driver package to access ISocketStorageDiscovery.

Reply:
why not put it in packages\drivers\odsp-driver-definitions

Reply:
If we end up putting it somewhere on our side, that's probably where it'll be. But the idea was that the provider interfaces would be useful for a Loop component to discover something about an `IDocumentServiceFactory`, and components depending on service-specific packages was a non-starter.

That said... I guess we could put them in odsp-driver-definitions and have the Loop app re-export them for the components to use from some package they already depend on? I'll explore that option with the Loop folks.